app/scripts/views/visits/index.js

```diff
@@ -23,7 +23,7 @@ define([
       this.lastVisitId = null;
       this.hasMoreVisits = true;
       this.loadingMoreVisits = false;
-      this.count = 100;
+      this.count = 50;
```

Comment:
might be nice to add one of those "defer image loading until scrolled into view" things at some point